lhotse/audio.py

```diff
@@ -218,6 +219,19 @@ def filter(self, predicate: Callable[[Recording], bool]) -> 'RecordingSet':
         """
         return RecordingSet.from_recordings(rec for rec in self if predicate(rec))
 
+    def split(self, num_splits: int, randomize: bool = False) -> List['RecordingSet']:
+        """
+        Split the ``RecordingSet`` into ``num_splits`` pieces of equal size.
```

Comment:
> into ``num_splits`` pieces of equal size

The comment is misleading and does not match the implementation.

It does not describe how it handles the case when `len(self) % num_splits != 0`. 

There is an option `drop_last` in PyTorch's dataloader and `drop_remainder` in Tensorflow's dataset.

The current implementation simply uses the semantic `drop_last=False` without
documenting it.

Reply:
I’ll fix it (soon)